tests/poll/delete.yml

```diff
@@ -0,0 +1,122 @@
+---
+db:
+  poll/1/meeting_id: 1
+
+
+cases:
+- name: poll
+  action: poll.delete
+  payload:
+    id: 1
+
+  cases:
+  - name: motion
+    db:
+      poll/1/content_object_id: motion/2
+    cases:
+    - name: Without perm
+      is_allowed: false
+
+    - name: With perm
+      permission: motion.can_manage
+      is_allowed: true
+
+  - name: assignment
+    db:
+      poll/1/content_object_id: assignment/2
+    cases:
+    - name: Without perm
+      is_allowed: false
+
+    - name: With perm
+      permission: assignment.can_manage
+      is_allowed: true
+
+  - name: other
+    cases:
+    - name: Without perm
+      is_allowed: false
+
+    - name: With perm
+      permission: agenda_item.can_manage
+      is_allowed: true
+
+- name: option
+  action: option.delete
+  db:
+    option/3/poll_id: 1
+  payload:
+    id: 3
+
+  cases:
+  - name: motion
+    db:
+      poll/1/content_object_id: motion/2
+    cases:
+    - name: Without perm
+      is_allowed: false
+
+    - name: With perm
+      permission: motion.can_manage
+      is_allowed: true
+
+  - name: assignment
+    db:
+      poll/1/content_object_id: assignment/2
+    cases:
+    - name: Without perm
+      is_allowed: false
+
+    - name: With perm
+      permission: assignment.can_manage
+      is_allowed: true
+
+  - name: other
+    cases:
+    - name: Without perm
+      is_allowed: false
+
+    - name: With perm
+      permission: agenda_item.can_manage
+      is_allowed: true
+
+
+- name: vote
+  action: vote.delete
```

Comment:
There is no public action `vote.delete`. Is this required for internal actions in the backend? If so: If I remember correctly no permission checks are done for internal actions.

Reply:
I think there is. 

Go into the backend repo and run:
```
make run-prod
```
Then you can open http://localhost:9002/health

There you can find a list of all known actions.

@normanjaeckel

Reply:
@FinnStutzenstein Norman says, this is not an internal action. Please talk with the backend team

Reply:
Wait for https://github.com/OpenSlides/openslides-backend/issues/458